exporters/prometheus/exporter_test.go

```diff
@@ -1006,7 +1006,7 @@ func TestExemplars(t *testing.T) {
 				}
 			}
 			require.NotNil(t, exemplar)
-			require.Equal(t, tc.expectedExemplarValue, exemplar.GetValue())
+			require.InDelta(t, tc.expectedExemplarValue, exemplar.GetValue(), 0.01)
```

Comment:
require.Equal is fine here as the exemplar value will be always exactly the same (it is set by a single recording call)